JobScheduler/JobScheduler/Jobs/GitHubRepositoryParserJob.cs

```diff
@@ -153,17 +34,21 @@ public override async Task Execute(DateTime signalTime, CancellationToken token)
             else
             {
                 var result = await ProcessRepositories(token);
-                //Console.WriteLine(result);
                 Deserialize(result);
             }
         }
 
         private void Deserialize(string stringJson)
         {
+            var date = new DateTime(2013);
             var list = JsonConvert.DeserializeObject<Generic[]>(stringJson);
-            foreach (var item in list)
+            var parseDate = list.Select(x => x.CreatedAt)
+                .TakeWhile(x => x.ToLocalTime() > date.ToLocalTime())
+                .OrderByDescending(x => x.ToLocalTime());
```

Comment:
что-то сложно очень получилось
```suggestion
            var repositories = list.Where(x => x.CreatedAt.Year > 2013).ToArray();
```